scss/tech.scss

```diff
@@ -49,3 +49,25 @@ $tech-logo-size: 100px;
   object-fit: fill;
   width: $tech-logo-size;
 }
+
+  @media only screen and (max-width: 1024px) {
+    .boxes {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        padding: 0;
+    }
+
+    .box {
```

Comment:
from this line till the end of the media query elements are indented 2 spaces more than they should